src/benchmarklib/benchmark_utils.cpp

```diff
@@ -269,6 +270,8 @@ EncodingConfig::EncodingConfig(ColumnEncodingSpec default_encoding_spec, Encodin
       type_encoding_mapping{std::move(type_encoding_mapping)},
       custom_encoding_mapping{std::move(encoding_mapping)} {}
 
+EncodingConfig EncodingConfig::unencoded() { return EncodingConfig{ColumnEncodingSpec{EncodingType::Unencoded}}; }
```

Comment:
The problem is that TCPC and TPCH have diverged in how the tables are created. And for TPCC, this option is needed because there is no flag. So if you want to have the TPCC tables unencoded, you would have to create `EncodingConfig{ColumnEncodingSpec{EncodingType::Unencoded}}` every time. I just wanted to provide a small convenience function to get that config.

Reply:
We should bring those two back together then, at least in terms of the external interface. Maybe @dimitri-schmidt could have a look at that when he implements the query part of TPC-C.

@lawben: Could you open an issue and briefly explain the divergence and what needs to be done?